src/test/java/uk/ac/ebi/eva/evaseqcol/controller/SeqColComparisonControllerIntegrationTest.java

```diff
@@ -0,0 +1,103 @@
+package uk.ac.ebi.eva.evaseqcol.controller;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+import org.springframework.web.client.RestTemplate;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import uk.ac.ebi.eva.evaseqcol.entities.SeqColLevelTwoEntity;
+import uk.ac.ebi.eva.evaseqcol.io.SeqColGenerator;
+import uk.ac.ebi.eva.evaseqcol.io.SeqColWriter;
+
+import java.io.IOException;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@Testcontainers
+public class SeqColComparisonControllerIntegrationTest {
+
+    @LocalServerPort
+    private int port;
+
+    private String baseUrl = "http://localhost";
+
+    @Value("${server.servlet.context-path}")
+    private String contextPath;
+
+    private final String COMPARISON_PATH = "/comparison";
+
+    private final String SEQCOL_A_DIGEST = "3mTg0tAA3PS-R1TzelLVWJ2ilUzoWfVq"; // seqCol test digest
+
+    private final String SEQCOL_B_DIGEST = "rkTW1yZ0e22IN8K-0frqoGOMT8dynNyE";
+
+    @Autowired
+    private SeqColWriter seqColWriter;
+
+    @Autowired
+    private SeqColGenerator seqColGenerator;
+
+    private static RestTemplate restTemplate;
+
+    @Container
+    static PostgreSQLContainer<?> postgreSQLContainer = new PostgreSQLContainer<>("postgres:15.2");
```

Comment:
Suggest changing this to 14.0 which is the EBI supported version at this time...